tests/sentinel/tests/00-base.tcl

```diff
@@ -31,6 +31,42 @@ test "Sentinel command flag infrastructure works correctly" {
     }
 }
 
+test "SENTINEL HELP output the sentinel subcommand help" {
+    assert_match "*SENTINEL <subcommand> *" [S 0 SENTINEL HELP]
+}
+
+test "SENTINEL MYID return the sentinel instance ID" {
+    assert_equal 40 [string length [S 0 SENTINEL MYID]]
+    assert_equal [S 0 SENTINEL MYID] [S 0 SENTINEL MYID]
+}
+
+test "SENTINEL RESET can resets the master" {
+     # can't actually do a reset now
+     assert_equal 0 [S 0 SENTINEL RESET foo]
```

Comment:
some of these tests are rather useless, even the coverage they give is a fake one (i.e. the command reply schema is covered, but the code inside the command, as well the actual reply value isn't really covered)

we can keep these here for now and improve them later (deleting these tests and creating new ones), but maybe it's easier to just add them to the skip list?

Reply:
i refresh the tests. now i think we can keep it